icu4j/main/core/src/main/java/com/ibm/icu/util/IslamicCalendar.java

```diff
@@ -1179,9 +1207,11 @@ private static int firstIslamicStartYearFromGrego(int year) {
         return year - 579 + shift;
     }
 
-    /*
+    /**
+     * @internal
```

Comment:
This is the "real" change here. This override method was added in
- https://github.com/unicode-org/icu/pull/3470